src/shared/types.ts

```diff
@@ -43,15 +44,17 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_EN_US',
     ianaTimezone: 'America/New_York',
     prospectTypes: [
+      ProspectType.COUNTS,
+      ProspectType.TIMESPENT,
+      ProspectType.RECOMMENDED,
       ProspectType.TOP_SAVED,
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
-      ProspectType.SYNDICATED_NEW,
+      ProspectType.DOMAIN_ALLOWLIST,
```

Comment:
is domain-allow list same as domain-allowlist new tab en-us? 
![Screen Shot 2023-05-02 at 10 59 38 AM](https://user-images.githubusercontent.com/12415584/235748927-54fe444a-1d45-45e4-91ff-32f4e65a38d7.png)

Reply:
redirected to doc questions

Reply:
Not sure about your question but we have one `DOMAIN_ALLOWLIST` type which is being used by new_tab_en_us, new_tab_de_de, new_tab_es_es, new_tab_fr_fr and new_tab_it_it :thinking: